Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.7
+FROM alpine:latest
```

Comment:
We should still nail the version, otherwise it might cause a problem in the future.

Reply:
done